modules/manage/pages/disaster-recovery/shadowing/failover-runbook.adoc

```diff
@@ -252,12 +462,48 @@ Document the time of failover initiation and completion, applications affected a
 
 If topics remain stuck after addressing these cluster health issues and you need immediate failover, you can force delete the shadow link to failover all topics:
 
+ifndef::env-cloud[]
+// tag::rpk-tab-force-delete[]
 [,bash]
 ----
 # Force delete the shadow link to failover all topics
+ifndef::env-cloud[]
 rpk shadow delete <shadow-link-name> --force
+endif::[]
+ifdef::env-cloud[]
+rpk shadow delete <shadow-link-name>
+endif::[]
+----
+// end::rpk-tab-force-delete[]
+endif::[]
+
+ifdef::env-cloud[]
+[tabs]
+======
+rpk::
++
+--
+include::manage:disaster-recovery/shadowing/failover-runbook.adoc[tag=rpk-tab-force-delete]
+
+`rpk shadow delete` force deletes the shadow link by default in Redpanda Cloud.
+--
+
+Cloud API::
++
+--
+[,bash]
+----
+curl -X DELETE https://api.redpanda.com/v1/shadow-links/<shadow-link-id> \
+  -H "Content-Type: application/json" \
+  -H "Authorization: Bearer ${RP_CLOUD_TOKEN}"
 ----
 
+The `DELETE /shadow-links/<shadow-link-id>` endpoint of the Control Plane API force deletes the shadow link by default in Redpanda Cloud.
+--
+======
+endif::[]
+
+[[force-delete-warning]]
```

Comment:
@c-julin @r-vasquez @simon0191 Do both delete operations (rpk & public cloud API) force delete by default when run in Cloud? https://deploy-preview-1498--redpanda-docs-preview.netlify.app/redpanda-cloud/manage/disaster-recovery/shadowing/failover-runbook/#topics-stuck-in-failing_over-state

Reply:
Yes